18F/midas
# Add user location and agency columns to users.csv

This addresses issue #717 

---

*Change Summary:* 
The two fields 'location' and 'agency' have been added to the list of fields to export on the `User` model. Since the fields don't actually come from the user model itself, this seems a bit awkward, so I added a comment to note where they're populated. Let me know if there's a more appropriate way to handle it.

The `user/export` handler now retrieves all tags tied to the users being exported. The tags are mapped onto the user objects, with the tag type (ex: 'location) being used as the key and the tag name (ex: 'Tempe, AZ') being used as the value.

*Testing Changes:*
I updated the existing functional test on the `export` feature. Since I wanted to test that the columns were actually being populated, I do some setup in the test to make a new tag and give a user the tag as their location.

*How to Test*
* Server tests should pass
* As an admin, go to the Administration view and select 'Manage Users'. Select the export button and view the `users.csv` file that downloads. You should see 9 columns, including 'location' and 'agency', and the two new columns should be populated for users that have a location and/or an agency shown on their profile.
